feedback/utils.py

```diff
@@ -1,6 +1,44 @@
 """Utilities for feedback app"""
 
+# feedbackblock/feedback/utils.py
+from urllib.parse import urlencode, urlparse, urlunparse
+from opaque_keys.edx.keys import UsageKey
+from django.conf import settings
```

Comment:
please follow the [style guide for imports ](https://docs.openedx.org/en/latest/developers/references/developer_guide/style_guides/python-guidelines.html#imports-order)
```suggestion
from urllib.parse import urlencode, urlparse, urlunparse

from django.conf import settings
from opaque_keys.edx.keys import UsageKey
```